src/index.html

```diff
@@ -3,11 +3,62 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com"
+    >
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet"
+    >
+    <link rel="stylesheet"
+          href="./styles/index.scss"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+
+      <img
+        src="images/imac.jpeg"
+        alt="iMac computer image"
+        class="card-image"
+      >
+
+      <h2 class="card-main-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card-subtitle-text">Product code: 195434</p>
+
+      <div class="card-stars"
+           data-qa="card"
+      >
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="stars-reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card-price-container">
+        <p class="card-price">Price:</p>
+        <p class="card-price-num">$2,199</p>
+      </div>
+
+      <a href="#"
+         class="card-button-buy"
+         data-qa="hover">
+         BUY
+      </a>
```

Comment:
```suggestion
      <a 
        href="#"
        class="card-button-buy"
        data-qa="hover"
      >
        BUY
      </a>
```
**every** attribute in the new line if there are 3 or more attributes
pay attention to indentation and `>` place